### PR TITLE
Fix scrolling issues

### LINKED ISSUE #2636: Scrolling in the list of messages misses messages
This is either a missuse of the Vue component for virtual lists or a bug in the component itself.

## How to test
- Create a new conversation as user A
- Add user B
- Send message "1" as user A
- Send message "2" as user B
- Send message "3" as user A
- Send message "4" as user B
- Send message "5" as user A
...
- Send message "16" as user B
- Scroll to the top of the list

### Expected result
The system messages about the conversation being created and user B being added are shown.

### Actual result
The system messages about the conversation being created and user B being added are not shown; message "1" is the first message shown (without date). Note that [the messages grouped by author](https://github.com/nextcloud/spreed/blob/5368f4a98a95d3193c59a03513668cdcee7acbca/src/components/MessagesList/MessagesList.vue#L119) and passed to the component contain the messages that are not shown.

Moreover, if the list is scrolled to the bottom and a new message is sent the list will not be updated. It is necessary to scroll back slightly for the new message to appear. If more messages are added alternatively (for example, until the message "30") scrolling up causes the messages to jump from message "15" to message "8" when close to the top.

### LINKED ISSUE #2681: Scrolling cannot continue and prevent loading any further message
On some specific conversation, I get old messages, first, but when I want to scroll down it flashes different messages but goes back to the previous state

Peek 07-01-2020 11-32

cc @nickvergessen @ma12-co

### PR DESCRIPTION
**Pull request by ma12-co**
Fixes #2636
Fixes #2681

The issues are fixed  by removing the virtual scroller

plus auto-scrolls the message list to the bottom when receiving new messages `&&`:
* the list was already scrolled to the bottom before receiving the new messages;
* the browser window is active.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,6 +39,7 @@
 		"vue-observe-visibility": "^0.4.6",
 		"vue-prevent-unload": "^0.2.3",
 		"vue-router": "^3.1.3",
+		"vue-scroll": "^2.1.13",
 		"vue-virtual-scroll-list": "^1.4.4",
```
Comment (nickvergessen) [inline package.json]: Remove as its not used anymore?

**Comment by nickvergessen** [comment]
Only thing I find confusing at the moment:
When you are not scrolled to the bottom and write a message, you jump to the bottom.
I think it's undesired?! I might want to reply to multiple individual messages before scrolling down?

File: package.json (hunk 1 above)
Comment (ma12-co) [reply package.json]: Nope, we do use it :) 
https://github.com/nextcloud/spreed/pull/2692/files#diff-1c90ff38a08209f9ebd4d05d1e43358eR68

**Comment by ma12-co** [comment]
> When you are not scrolled to the bottom and write a message, you jump to the bottom. 

This is on purpose. I think it's as it should be

**Comment by danxuliu** [comment]
> This is on purpose. I think it's as it should be
> I quickly checked and every other chat app that I have on my phone/laptop does that!

We are not every other app! :-P I agree with @nickvergessen, I think that scrolling to bottom should not be forced when entering a new message. I think that it should be done only if the list is already at the bottom (and, if I am not mistaken, that is how it worked in Talk 7).

In any case as there are different views on that I would leave it for another pull request, as [that behaviour was already there](https://github.com/nextcloud/spreed/blob/cac8dc7193d38bfae1d38c265fcc75c55ca79078/src/components/NewMessageForm/NewMessageForm.vue#L168-L171) and is not caused by the changes in this one.

**Comment by ma12-co** [comment]
Well every other app includes us in the form of talk android :)

**Comment by nickvergessen** [comment]
DCO failed?

File: package.json (hunk 1 above)
Comment (nickvergessen) [reply package.json]: I commented on vue-virtual-scroll-list....

**Pull request merged**